src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -14,13 +18,62 @@
   "v3")
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Private
+
+(defn class-info-from-jruby->class-info-for-json
+  "Convert a class info map received from the jruby service into an
+  appropriate map for use in serializing an environment_classes response to
+  JSON"
+  [info-from-jruby environment]
```

Comment:
As discussed in meatspace, I'd be fine with a schema as simple as `{schema/Keyword schema/Any}`, or whatever granularity is reasonable.  We don't need to cover all of the variable use cases super-explicitly, but it would make the code more readable to just even know that the input was a map and a string, and the output was a map.

Also agree about adding more tests, and I think maybe we talked about adding that as a line-item in our intern spreadsheeit.